app/models/reducers/summary_statistics_reducer.rb

```diff
@@ -0,0 +1,175 @@
+module Reducers
+  class SummaryStatisticsReducer < Reducer
```

Comment:
This class will need to be added to https://github.com/zooniverse/caesar/blob/master/app/models/reducer.rb#L4 and https://github.com/zooniverse/caesar/blob/d028d962b807f44219185e1002b47dbd18fe2425/app/models/workflow/convert_legacy_reducers_config.rb#L23.

Reply:
done!